src/components/Shared/index.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+export const Highlight = ({children, color}) => (
```

Comment:
i would create mapper with colors and use key as parameter to reuse it in different places without think about HEX color name.

f.e.
`const map = { post: 109049 }`
use in parameters:
`color="post"`